br/cmd/br/cmd.go

```diff
@@ -107,6 +110,23 @@ func AddFlags(cmd *cobra.Command) {
 	_ = cmd.PersistentFlags().MarkHidden(FlagRedactLog)
 }
 
+func calculateMemoryLimit(memleft uint64) uint64 {
+	const halfGB = uint64(512 * 1024 * 1024)
```

Comment:
I feel like it would be better to put it into some util file for reuse.

Reply:
done. Reuse the `tidb/pkg/util/size.MB`.